content/bn/docs/reference/glossary/annotation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: এনোটেশন
+id: annotation
+date: 2018-04-12
+full_link: /docs/concepts/overview/working-with-objects/annotations
+short_description: >
+  অবজেক্টগুলির উপর অনিয়মিতভাবে অপরিচিত মেটাডেটা সংযুক্ত করার জন্য ব্যবহৃত একটি কী-মান জোড়।
+aka: 
```

Comment:
maintain proper line break

![image](https://github.com/kubernetes/website/assets/155321064/19c790bc-cc5c-4741-93b7-17cb7870d519)

Reply:
(no need to have a line break there, but it's OK to leave it in)

Reply:
fixed this.

Reply:
@sftim @Ashfinn  No need here, but we try to maintain proper line breaks everywhere. It becomes hard to update or review big files when we don't have proper line breaks as in the English version.